packages/codegen-ui-react/lib/forms/form-renderer-helper.ts

```diff
@@ -194,11 +199,17 @@ export const addFormAttributes = (component: StudioComponent | StudioComponentCh
       attributes.push(
         factory.createJsxAttribute(
           factory.createIdentifier('value'),
-          factory.createJsxExpression(undefined, getCurrentValueIdentifier(componentName)),
+          factory.createJsxExpression(
+            undefined,
+            getCurrentValueIdentifier(fieldConfig.mappedVariableName || componentName),
+          ),
         ),
         factory.createJsxAttribute(
           factory.createIdentifier('ref'),
-          factory.createJsxExpression(undefined, factory.createIdentifier(`${lowerCaseFirst(componentName)}Ref`)),
+          factory.createJsxExpression(
+            undefined,
+            factory.createIdentifier(`${lowerCaseFirst(fieldConfig.mappedVariableName || componentName)}Ref`),
```

Comment:
We just removed the `lowerCaseFirst` functionality in @awinberg-aws 's PR. Can you merge his in and make sure everything still passes?

Reply:
Yeah, I was going to wait for his PR to merge.